corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -35,7 +35,7 @@
 
 @register.simple_tag
 def render_form_xml(form):
-    xml = form.get_xml()
+    xml = form.get_xml().encode('utf-8')
```

Comment:
It seems that when we turn the same XML string into an etree element, [we check its type first][1]. To be consistent we should probably go:
```python
    xml = form.get_xml()
    if isinstance(xml, unicode):
        xml = xml.encode('utf8', errors='replace')
```

  [1]: https://github.com/dimagi/commcare-hq/blob/df78fea8ada7124042e3bb9cb6ce166b8212c298/corehq/ex-submodules/couchforms/models.py#L275-L275